lib/resources/r4/related_person_patch.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: Related Person
+field_name_base_url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson
+operations:
+  - name: add-relationship
+    path: /relationship/-
+    operation: add
+    type: codeableConcept
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
+    description: The nature of the relationship between a patient and the related person.
+    example: |
+      [
+        {
+          "op": "add",
+          "path": "/relationship/-",
+          "value":{
+            "coding": [
+              {
+                "system": "https://fhir.cerner.com/d242a518-4074-4bd4-a3a6-adfe0c5c1c51/codeSet/351",
+                "code": "1152"
+              }
+            ]
+          }
+        }
+      ]
+
+    note: >
+      <ul>
+        <li>The <code>value.coding</code> must contain exactly one proprietary system.</li>
+        <li>The <code>value.coding</code> cannot contain a standard system.</li>
```

Comment:
We need to note that if you add a duplicate relationship you wont see it on a read
https://teams.microsoft.com/l/message/19:c52e80fcd36c4ec3914f1ba0af8370e1@thread.skype/1607444522288?tenantId=fbc493a8-0d24-4454-a815-f4ca58e8c09d&groupId=dce636bf-7e25-47d2-a255-c5e7f16951da&parentMessageId=1607444522288&teamName=SMART%2FFHIR&channelName=HDX%20RevCycle%20Collaboration&createdTime=1607444522288

Reply:
I think we should add this under read/search workflow. @alexbeckner1 What do you think?

Reply:
Added here: https://github.com/cerner/fhir.cerner.com/pull/527/commits/61caf398efa0d2303c76a492f6b7c0b804548843#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR31

Reply:
Made a comment to add this note to read/search documentation as well. https://github.com/cerner/fhir.cerner.com/pull/526#issuecomment-747359043